src/helpers/HelperAssert.sol

```diff
@@ -22,6 +22,10 @@ abstract contract HelperAssert is HelperBase {
         }
     }
 
+    function createErrorMessage(string memory aStr, string memory bStr, string memory operator, string memory reason) private pure returns (bytes memory) {
```

Comment:
let's move this one to the bottom because it is a helper function

Reply:
make it `internal` instead of `private` so we can use it in child contracts as well (like the foundry tests)

Reply:
i wonder if "error message" is the best name, perhaps we should call it `createAssertFailMessage`? what do you think? :)

Reply:
Lets's do it!!